src/Nager.Date/HolidaySystem.cs

```diff
@@ -143,6 +145,33 @@ public static class HolidaySystem
         /// </remarks>
         public static string LicenseKey = null;
 
+        private static void CheckLicense(string licenseKey)
+        {
+            if (string.IsNullOrEmpty(licenseKey))
+            {
+                throw new NoLicenseKeyException();
+            }
+
+            var licenseKeyConfiguration = new LicenseSystem.LicenseKeyConfiguration
+            {
+                Part1 = "99FFD01352B56E4AAFE51D73D793EF7FDDFB67F03BD8E60600F77C401B19C645",
+                Part2 = "FCDEB16976C7CC3D4AD3296F6408F89A48BF76B49E0B7D0857BDF242DA97C4BE"
+            };
+
+            var licenseKeyValidator = new LicenseSystem.LicenseKeyValidator(licenseKeyConfiguration);
+            if (!licenseKeyValidator.Validate(licenseKey, out var licenseInfo))
+            {
+                throw new NoLicenseKeyException();
+            }
+
+            if (licenseInfo.ValidUntil < DateTime.Today)
+            {
+                throw new NoLicenseKeyException();
+            }
```

Comment:
Just to reduce complexity

```suggestion
            if (!licenseKeyValidator.Validate(licenseKey, out var licenseInfo) || licenseInfo.ValidUntil < DateTime.Today)
            {
                throw new NoLicenseKeyException();
            }
```